src/cursor/cur_std.c

```diff
@@ -567,6 +579,36 @@ __wt_cursor_get_valuev(WT_CURSOR *cursor, va_list ap)
     API_END_RET_STAT(session, ret, cursor_get_value);
 }
 
+/*
+ * __wt_cursor_get_raw_key_value --
+ *     WT_CURSOR->get_raw_key_value default implementation
+ *
+ */
+int
+__wt_cursor_get_raw_key_value(WT_CURSOR *cursor, WT_ITEM *k, WT_ITEM *v)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    CURSOR_API_CALL(cursor, session, get_value, NULL);
+
+    if (!F_ISSET(cursor, WT_CURSTD_VALUE_EXT | WT_CURSTD_VALUE_INT))
```

Comment:
Does having a value set always imply that there is a key set as well? i.e: Does this also need to check `WT_CURSTD_KEY_XX`?

Reply:
Good point!

Reply:
Check added!